ocs_ci/deployment/hosted_cluster.py

```diff
@@ -660,6 +745,25 @@ def create_host_inventory():
     logger.info("Created InfraEnv.")
 
 
+def get_onboarding_token_from_secret(secret_name):
```

Comment:
There could be some parameter with retry and wait so it can be used without the wait.

Reply:
we have retry few lines bellow
`secret_obj = ocp_obj.get(retry=6, wait=10, silent=True)`